alerter.c

```diff
@@ -1,16 +1,9 @@
 #include <stdio.h>
 #include <assert.h>
+#include "stubNwAlert.h"
 
 int alertFailureCount = 0;
 
-int networkAlertStub(float celcius) {
-    printf("ALERT: Temperature is %.1f celcius.\n", celcius);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
-}
-
 void alertInCelcius(float farenheit) {
     float celcius = (farenheit - 32) * 5 / 9;
     int returnCode = networkAlertStub(celcius);

```

Comment:
Good approach to move stub code to a different file. Next improvement is to abstract the use of stub in production code by using either an interface or function pointer